packages/interface/src/client/useAuthorization.ts

```diff
@@ -59,11 +59,11 @@
       (authorization) => {
         window.localStorage.setItem(
           "authx-interface.authorization",
-          `${authorization.id}:${authorization.secret}`
+          `${authorization.id}:${authorization.secret}`,
```

Comment:
The alert seems to think the `authorization.secret` is something more than the "session" token, and their recommendation is more or less what's already implemented – access from this value can be revoked, and there is no residual sensitivity.

However, we may benefit from switching to http-only cookies from localStorage, to reduce the risk of credential theft due to XSS, etc. Leaving open because of this.